source/agora/cli/SendTxProcess.d

```diff
@@ -217,65 +217,24 @@ public int sendTxProcess (string[] args, ref string[] outputs,
 /// Test of send transaction
 unittest
 {
-    class TestCLINode : API
+    import std.typecons;
+    class TestCLINode : BlackHole!API
```

Comment:
```suggestion
    // Any non-explicitly implement API function will do nothing and return `typeof(return).init`
    class TestCLINode : BlackHole!API
```

Just to make it clear what happens, `BlackHole` is not the most popular function around here.

Reply:
Unfortunately `WhiteHole` is broken: https://issues.dlang.org/show_bug.cgi?id=20232 ;)